src/main/java/com/treasuredata/client/TDClient.java

```diff
@@ -993,4 +1010,16 @@ else if (!jobRequest.getResultOutput().isEmpty()) {
             return Optional.absent();
         }
     }
+
+    @Override
+    public TDImportResult importFile(String databaseName, String tableName, String format, File file)
+    {
+        return doPut(buildUrl(String.format("/v3/table/import/%s/%s/%s", databaseName, tableName, format)), ImmutableMap.of(), file, TDImportResult.class);
+    }
+
+    @Override
+    public TDImportResult importFile(String databaseName, String tableName, String format, File file, String id)
```

Comment:
We need to explain what is `id` here, because to understand the meaning of this parameter, I need to check the code of td-api.  This id will be used for duplicate elimination in the server side.